server/views/partials/list_filter.njk

```diff
@@ -0,0 +1,86 @@
+{% call govukFieldset({
+legend: {
+text: "Filter clients",
```

Comment:
Indentation of this file looks a little strange?

Reply:
updated